themes/hugo-bulma-blocks-theme/layouts/shortcodes/tabs.html

```diff
@@ -5,7 +5,8 @@
 
 {{ with index .Site.Data.conf }}
 	{{ $tab1 = replace $tab1 "|ltrversion|" .ltrversion }}
-	{{ $tab2 = replace $tab2 "|ltrversion|" .ltrversion }}
+	{{ $tab2 = replace $tab2 "|version|" .version }}
```

Comment:
```suggestion
	{{ $tab2 = replace $tab2 "|ltrversion|" .ltrversion }}
	{{ $tab2 = replace $tab2 "|version|" .version }}
```
How would the above work? In february, when 3.40 becomes LTR, 3.34 goes to archives so we will move back to 3 tabs mentioning ltrversion only. Would the above avoid us touching again this file at that moment?

Reply:
Sorry @DelazJ , I missed your suggestion. I will submit a new PR afterwards.
If we apply this, we will have either the raw |version| text or a duplicated LTR version tab. I think there should be a better way to manage this without touching any related files in the future.

![image](https://github.com/user-attachments/assets/6c5c6458-c520-476d-b3d9-43a85a3ba973)

Reply:
Applied with #471